GreenField-Org/ultracrewapi
# 18 helmet

## Server Verification locally
- `git pull` from main
- `git checkout 18-helmet`
- `npm ci`
- `npm run start`
- open browser and go to http://localhost:8080
- Verify you see Hello World


Update the following api call to test the create user endpoint:

http://localhost:8080/api/user?username=vigil_test&firstName=Julian&lastName=Vigilante&middleInitial=H&password=welcome&dob=12/12/1990&weight=150&country=USA&address=111 Welcome Terrace Albany, NY 12202&email=vigil_test@mail.com&phoneNum=+15555555555&userTier=1

### Steps to Test on Browser
- put the updated endpoint in the browser 
- Hit Enter
- Verify 200 response

### Steps to Test on Client
- Open client with http://localhost:8080/api/user
- Fill in the necessary parameters
- Click send
- Verify 200 status
